jenkins_agent_snap/snapcraft.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: jenkins-agent
+summary: Jenkins agent snap
+description: Agent for jenkins
+version: "1.0.0"
+base: core22
```

Comment:
Why not core24?

Reply:
It doesn't seem to be available at the moment